packages/apollo-voyager-tools/src/errors/SyncServerError.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQLError } from 'graphql'
+const prefix = 'AgSync:'
+const VALIDATION_TYPE = prefix + 'Validation'
+
+/**
+ * Represents server side error
+ */
+export class SyncServerError extends GraphQLError {
```

Comment:
Perhaps this could be `VoyagerServerError` ? @wtrocki any opinions on this? I don't really mind either way. The word `Sync` is not really used anywhere else in the codebase though.

Reply:
Refactored this to error. I have added more info into commits so motivation is clear.
@StephenCoady Freel free to review that after you are back.